src/main/java/org/apache/commons/collections4/bidimap/AbstractDualBidiMap.java

```diff
@@ -620,7 +617,7 @@ public void remove() {
         private static final long serialVersionUID = 4040410962603292348L;
 
         /**
-         * Constructs a new view of the BidiMap.
+         * Constructs a new view of the BidiMap
```

Comment:
I think we have some other statements in the code base that end in a period. Any specific reason to remove them? If not, I think it would be simpler to just leave them as-is.

Reply:
-1: A sentence ends with a period.